eng/Signing.props

```diff
@@ -87,4 +82,44 @@
     <ItemsToSign Remove="$(ArtifactsPackagesDir)**\*tests*.nupkg" />
```

Comment:
As you are already at it, you can also remove the above line as we removed support for post build signing in Arcade.

Reply:
We're still using post build signing on main, just not in release branches.

Reply:
Post build signing doesn't do anything anymore: https://github.com/dotnet/arcade/commit/d42c4892c6f3cd2fc28cf1ab0e7f7baa3f49d393